en/docs/deploy/monitor/elk-mask-sensitive-information-in-logs.md

```diff
@@ -34,12 +31,8 @@ To configure this feature, follow the instructions given below.
     output.logstash:
         ...
     ```
-3. Edit the function `process(event)` to mask the fields inside the event. Sample is given to replace `username` field
-   in the event to the word `MASKED`. This is done
-   through [JavaScript replace() function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace).
-
-!!! note    
-      There can be a performance impact when using this
-      feature with many masking patterns since each log line is matched with
-      each of the patterns. Hence, it is recommended to only use the most necessary patterns.
     
+
+!!! note
+      As the number of masking patterns increases, performance will be affected since each log line is matched with each of the patterns. It is recommended to only use the most necessary patterns.
```

Comment:
```suggestion
      As the number of masking patterns increases, performance will be affected since each log line is matched with each pattern. It is recommended to only use the most necessary patterns.
```